federation/cluster/common.sh

```diff
@@ -92,8 +92,6 @@ function create-federation-api-objects {
     #We will use loadbalancer services where we can
     export FEDERATION_API_NODEPORT=32111
     export FEDERATION_NAMESPACE
-    export FEDERATION_NAME="${FEDERATION_NAME:-federation}"
```

Comment:
My original goal was to get rid of these scripts and this was in line to that goal. We probably need this back given our new goal of having two scripts in parallel.